change-beta/@azure-communication-react-ad25f56f-cdbf-483e-96ec-846c3f466f48.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Update nodeJS wording to require version 14",
```

Comment:
Instead of requiring node 14, I think we should ensure we are working with LTS always.
Instead can we update the `rush.json` to something like
```json
"nodeSupportedVersionRange": ">=12.13.0 <13.0.0 || >=14.15.0 <18.0.0",
```

@dmceachernmsft do we still have outstanding issues with node 18 or were those fixed?

Reply:
The rush tool _only_ supports version 14 in the current setup. 
`"nodeSupportedVersionRange": ">=12.13.0 <13.0.0 || >=14.15.0 <15.0.0",`
This limits the rest of the system to 14 also, does it not?

Reply:
Oh yep great point it should filter out odd numbers - but I think it should JustWork if we update that version range. We should also drop anything below 14 as thats deprecated.
Donald was doing some testing with node 18 so keen to know if we can flag that as supported also

Reply:
Maybe this is something to discuss internally before putting in a PR. I would have thought moving Node versions could have bigger impact - and more testing first.